### ISSUE TITLE
Accumulator problem

### ISSUE THREAD

**Issue by MariaPettersson** [Hopsan/hopsan#1794]
Failing accumulator in standard library?
The pressure should vary around HighPstart pressure (40 bar) due to mtion of moving wall.
Using the standard acc show variation at higher pressure.  Why?

**To Reproduce**
Run the attached model, and check the scope

![problem](https://user-images.githubusercontent.com/44434853/48051896-b4445d00-e1a6-11e8-9558-d7c53eb35755.JPG)

**Hopsan version (please complete the following information):**
*Version information is shown in the message widget when HopsanGUI is started*
 - OS: [ Windows 10]
 - Version: [e.g. HopsanCore 64-bit, Version: 2.9.0.20180912.0739
                       HopsanGUI 64-bit, Version: 2.9.0.20180915.2229]
 -

**Comment by robbr48** [comment]
I don't think there is an attached model.

What do you think @petkr14?

**Comment by MariaPettersson** [comment]
Best regards,
Maria Pettersson Ph.D
Senior Specialist Rock Drills
R&D Rock Drills and Rotation Units
________________________________
Epiroc
Rock Drill Product Line
Epiroc Rock Drills AB
Visitor's address: Klerkgatan 21
702 25 Örebro, Sweden
Postal address: SE-701 91 Örebro
Phone: +46196707330
E-mail: maria.pettersson@epiroc.com<mailto:maria.pettersson@epiroc.com>
http://www.epiroc.com/
Follow us at: Facebook<https://www.facebook.com/epirocgroup/> - LinkedIn<https://www.linkedin.com/company/16268827/> - Twitter<https://twitter.com/epirocgroup> - YouTube<https://www.youtube.com/channel/UCQLTt2pZVLqK1nVhjCopgeQ> - Instagram<https://www.instagram.com/epirocgroup>
________________________________
United in performance. Inspired by innovation.

**Comment by peterNordin** [comment]
I do not think that you can attach/upload arbitrary files, uploading pictures seem to work though.
You have to email the model directly to Robert I think

**Comment by peterNordin** [comment]
Also remember that when you replay to an email from GitHub, that email will be posted to the Issue thread and be visible to ALL. So make sure that no potentially "confidential" information is included i any such email.

**Comment by peterNordin** [comment]
Also bots may harvest your email address for spam purposes (I deleted your email reply for this reason) it did not contain any useful data, the attachment had been stripped out.

**Comment by robbr48** [comment]
I believe you can use zip files for arbitrary attachments:
https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/
